chef/cookbooks/opendaylight/attributes/default.rb

```diff
@@ -17,4 +17,4 @@
 # limitations under the License.
 #
 
-default[:opendaylight][:logfile] = "/dev/null"
+default[:opendaylight][:features] = "odl-restconf odl-l2switch-switch odl-dlux-core odl-ovsdb-openstack"
```

Comment:
It would be better to have subsections for features: (for eg: 

```
features: {
    restconf:
      enabled: true
    l2switch:
      switch: true
    dlux: 
       core: true
       sfc: false
  ovsdb:
     openstack: true
     southbount: false
}
```
This structure will help us add specific configurations for each feature set.
Imagine this line with 15+ features in one line.